tests/utils/utils.py

```diff
@@ -0,0 +1,8 @@
+import typing
+from typing import List
+
+def assert_lines_match(actual : List[str], expected : List[str]):
+    for i in range(0, max(len(actual), len(expected))):
```

Comment:
`zip`

also, before the for, compare the `len`s (I assume that wrong number of lines is a problem)

Reply:
Actually, I would like to leave it as is... I understand that this could be optimized, but my intent is to actually make it ignore empty lines and such, to focus the match on the meaningful content. Zip goes against this idea...

Reply:
https://docs.python.org/3/library/itertools.html#itertools.zip_longest

Reply:
It may cover the existing behaviour, but it does not address the future intent. I consider this a premature optimization.

Reply:
I don't get it - as far as I understand zip_longest does exactly the same as the code here